packages/icons/scripts/build.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs-extra'
+import path from 'path'
+import { exec } from 'child_process'
+import os from 'os'
+
+async function build () {
+  const distPath = path.join(__dirname, '../dist')
+  const assetsPath = path.join(__dirname, '../assets')
+  const iconsPath = path.join(distPath, 'icons')
+  const iconsetPath = path.join(distPath, 'cypress.iconset')
+
+  await fs.remove(distPath)
+  await fs.mkdir(iconsPath, { recursive: true })
+
+  await exec('yarn tsc -p ./tsconfig.build.json')
+
+  if (os.platform() === 'darwin') {
```

Comment:
Not related to this PR, but does this platform specific build have any effect on what the binary expects? When we build the Cypress binary for windows, will it have `dist/icons/cypress.icns` available and is it needed? Maybe @elevatebart has an idea

Reply:
@ZachJW34, the answer is no and it does not need it. the only moment where we "need" the icns file is when we package the electron app for mac. `icns` is the mac specific icon format that contains multiple definitions. Since this electron app can only be built and signed on Mac, I am not afraid of this happening.